tests/search/range/exact/test.sd

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+search test {
+  document test {
+     field i1 type int {
+         indexing: attribute | summary
+         attribute: fast-search
+         match:exact
+     }
+     field f1 type int {
+         indexing: attribute | summary
+         attribute: fast-search
+         match:exact
+     }
+     field m1 type int[] {
```

Comment:
As far as I know, this syntax is not allowed. Should be _array<int>_.

Reply:
I guess it is now the only undocumented syntax that works....